pkg/analytics/tracking_utils.go

```diff
@@ -5,30 +5,34 @@ import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"go.uber.org/zap"
 	"math"
 	"net/http"
 	"time"
 
 	"github.com/klothoplatform/klotho/pkg/core"
 )
 
-var kloServerUrl = "http://srv.klo.dev"
+const kloServerUrl = "http://srv.klo.dev"
 
 type AnalyticsFile struct {
 	Id string
 }
 
-func SendTrackingToServer(bundle *Client) error {
-	postBody, _ := json.Marshal(bundle)
+func (t *Client) Send(payload *Payload) {
+	postBody, _ := json.Marshal(payload)
 	data := bytes.NewBuffer(postBody)
-	resp, err := http.Post(fmt.Sprintf("%v/analytics/track", kloServerUrl), "application/json", data)
-	if err != nil {
-		return err
+	url := t.serverUrlOverride
+	if url == "" {
+		url = kloServerUrl
 	}
+	resp, err := http.Post(fmt.Sprintf("%v/analytics/track", url), "application/json", data)
```

Comment:
I think `%s` is a better fit for this than `%v` since we specifically want the `.String()` method which gives the correct value. It ends up being the same in this case, but IMO it's clearer that you're not putting in like `url.URL{Scheme: "http", ...}`

Reply:
Good call. It was `%v` before and I didn't revisit that, but I agree `%s` is clearer.